spec/g10_certification_test_kit/bulk_data_group_export_spec.rb

```diff
@@ -0,0 +1,321 @@
+    end
+
+    it 'passes if client can not $export without authorization' do
+      stub_request(:get, "#{bulk_server_url}/Group/#{group_id}/$export")
+        .to_return(status: 401)
+
+      result = run(runnable, base_input)
+      expect(result.result).to eq('pass')
+    end
+  end
+
+  describe '[Bulk Data Server returns "202 Accepted" and "Content-location" for $export] test' do
+    let(:runnable) { group.tests[3] }
+
+    it 'fails when server does not return "202 Accepted"' do
+      stub_request(:get, "#{bulk_server_url}/Group/#{group_id}/$export")
+        .with(headers: { 'Authorization' => "Bearer #{bearer_token}" })
+        .to_return(status: 401)
+
+      result = run(runnable, base_input)
+
+      expect(result.result).to eq('fail')
+      expect(result.result_message).to eq('Bad response status: expected 202, but received 401')
+    end
+
+    it 'fails when server does not return "Content-Location" header' do
+      stub_request(:get, "#{bulk_server_url}/Group/#{group_id}/$export")
+        .with(headers: { 'Authorization' => "Bearer #{bearer_token}" })
+        .to_return(status: 202)
+
+      result = run(runnable, base_input)
+
+      expect(result.result).to eq('fail')
+      expect(result.result_message).to eq('Export response headers did not include "Content-Location"')
+    end
+
+    it 'fails when no value for "Content-Location" header' do
+      stub_request(:get, "#{bulk_server_url}/Group/#{group_id}/$export")
+        .with(headers: { 'Authorization' => "Bearer #{bearer_token}" })
+        .to_return(status: 202, headers: { 'content-location' => nil })
+
+      result = run(runnable, base_input)
+
+      expect(result.result).to eq('fail')
+      expect(result.result_message).to eq('Export response headers did not include "Content-Location"')
+    end
+
+    it 'passes when server returns both "202 Accepted" and "Content-location"' do
+      stub_request(:get, "#{bulk_server_url}/Group/#{group_id}/$export")
+        .with(headers: { 'Authorization' => "Bearer #{bearer_token}" })
+        .to_return(status: 202, headers: { 'content-location' => polling_url })
+
+      result = run(runnable, base_input)
+
+      expect(result.result).to eq('pass')
+    end
+  end
+
+  describe '[Bulk Data Server returns "202 Accepted" or "200 OK" for status check] test' do
+    let(:input) { base_input.merge(polling_url: polling_url) }
+    let(:runnable) { group.tests[4] }
+    let(:headers) { { 'content-type' => 'application/json' } }
+    let(:incomplete_status_response) do
+      status_response_json = JSON.parse(status_response)
+      status_response_json.delete('transactionTime')
+      status_response_json.to_json
+    end
+
+    it 'skips when polling_url is not provided' do
+      result = run(runnable)
+
+      expect(result.result).to eq('skip')
+      expect(result.result_message).to eq('Server response did not have Content-Location in header')
+    end
+
+    # NOTE: Test commented out as it takes three minutes to fail
```

Comment:
Check out TimeCop, it's how you deal with these situations: https://github.com/travisjeffery/timecop